website/linkinator.config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "recurse": true,
+  "silent": true,
+  "skip": [
+    ".(json|js|png|jpg|svg|css)(\\?(.*)+)?$",
```

Comment:
I included this rule to exclude check for static assets (which are handled by Gatsby anyway).